scripts/dockerfiles/Dockerfile.init-debug-s3

```diff
@@ -0,0 +1,47 @@
+FROM amazon/aws-for-fluent-bit:init-debug-base
+COPY ./scripts/dockerfiles/Dockerfile.init-debug-base /Dockerfile.2.init-debug-base
+
+ENV S3_BUCKET ""
+ENV S3_KEY_PREFIX issue
+
+RUN yum install -y unzip zip curl gdb
+
+WORKDIR /var/tmp
+RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-$(arch).zip" -o "awscliv2.zip"
+RUN unzip awscliv2.zip
+RUN ./aws/install
+RUN rm awscliv2.zip
+
+RUN yum remove unzip -y \
+    && yum clean all \
+    && rm -rf /var/cache/yum
+
+# For build efficiency, copy in the Fluent Bit config file again at this stage
+COPY fluent-bit.conf \
+     /fluent-bit/etc/
+
+COPY ./scripts/core_uploader.sh /
+RUN chmod +x /core_uploader.sh
+
+# Run Fluent Bit from the cores-out folder and collect crash symbols
+# Move symbols to /cores folder after processing
+RUN mkdir /cores-out
+WORKDIR /cores-out
+
+# Upload coredumps to s3 on shutdown
+# customer must set BUCKET env var
+
+# Only last CMD command will be executed, automatically replaces the original entrypoint
+CMD echo "AWS for Fluent Bit Container Image Version `cat /AWS_FOR_FLUENT_BIT_VERSION` - Init Debug Image with S3 Core Uploader"; \
+    if [ "$S3_BUCKET" == "" ]; then \
+        echo "Note: Please set S3_BUCKET environment variable to your crash symbol upload destination S3 bucket"; \
+    fi; \
+    if [ "$S3_KEY_PREFIX" == "issue" ]; then \
+        echo "Note: Please set S3_KEY_PREFIX environment variable to a useful identifier - e.g. company name, team name, customer name"; \
+    fi; \
+    /init/fluent_bit_init_process; \
+    chmod +x /init/invoke_fluent_bit.sh; \
+    export RANDOM_ID_VALUE=$(($RANDOM%99999))$(($RANDOM%99999))$(($RANDOM%99999)); \
+    echo "RANDOM_ID is set to $RANDOM_ID_VALUE"; \
+    env RANDOM_ID=$RANDOM_ID_VALUE /init/invoke_fluent_bit.sh; \
```

Comment:
why create and echo the random ID on startup? This will confuse user. We should only print this when we crash??

Reply:
It's because you can use the random id in your Fluent Bit configuration ie send logs to a cloudwatch log  stream with the random id. Then you have a way of connecting a fluent bit instance with a specific log stream

Reply:
hmmm ok... if this is going to be a new feature.. we should document it somewhere... it is just for debug images or for prod release too?

Reply:
It's just for the debug images, not in prod.